packages/terra-table/src/subcomponents/_Cell.jsx

```diff
@@ -55,7 +55,7 @@ const Cell = ({
   ...customProps
 }) => {
   const cellClassNames = cx('cell');
-  const contentClass = !removeInner ? cx('container') : undefined;
+  const contentClass = !removeInner ? cx('container') : cx('content-width');
```

Comment:
```suggestion
  const contentClass = cx([{
     'container': !removeInner,
     'content-width': removeInner,
  });
```

Reply:
If we do Dave's comment, this would not be needed.